### ISSUE TITLE
class RN()，var方差计算公式问题

### ISSUE THREAD

**Issue by ZiyangBo** [jinwan1994/SG-ShadowNet#6]
在class RN(nn.Module):这里是不是存在着设计错误

    def get_foreground_mean_std(self, region, mask):
        sum = torch.sum(region, dim=[2, 3])     # (B, C) 
        num = torch.sum(mask, dim=[2, 3])       # (B, C) 
        mu = sum / (num + self.eps) 
        mean = mu[:, :, None, None]

        var = torch.sum((region + (1 - mask)*mean - mean) ** 2, dim=[2, 3]) / (num + self.eps)
        var = var[:, :, None, None]
        return mean, torch.sqrt(var+self.eps)

按照论文书写公式：

![Image](https://github.com/user-attachments/assets/4572c820-b949-4d4e-9e99-197921786d7a)
var的计算是不是有问题
应该是
var = torch.sum(mask * (region - mean) ** 2, dim=[2, 3]) / (num + self.eps)
